jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile

```diff
@@ -7,7 +7,8 @@ rollingUpgradePipeline(
     params: params,
 
     backend: 'gce',
-    base_versions: ['2019.1', '3.2'],
+    // Version 3.1 is added to cover the issue #5621: upgrade from 3.1 to 3.2 fails
+    base_versions: ['2019.1', '3.2', '3.1'],
```

Comment:
Do you want to test upgrades from all those version to master?

- upgrade from 3.1 to master
- upgrade from 3.2 to master


Upgrade from 3.1 to 3.2 had been covered in jenkins file of SCT branch-3.2
- https://github.com/scylladb/scylla-cluster-tests/blob/branch-3.2/jenkins-pipelines/rolling-upgrade-centos7.jenkinsfile#L10

Reply:
This issue was found during upgrade between 3.1 to 3.2. If it will cover, so it's good enough. Rolled back this change